README.md

```diff
@@ -55,9 +55,9 @@ It is recommended that you use `virtualenv` (especially for `conda` users).
   * Installing `virtualenv` : ` sudo pip install virtualenv`
   * `virtualenv venv --python=python3`
   * `. venv/bin/activate`
-  * `pip3 install moboff`
+  * `python setup.py install`
```

Comment:
Does it make sense it to write `python3` instead of `python`, since if a user skips the virtual env part, it could result in some breaking.

Reply:
I think the user should be able to write any one of them which is appropriate to the system. On some systems, `python` may imply python 3 versions so we should leave the user to edit accordingly. Also, as the instruction mentions to install within a virtual environment, I think writing `python` is better. 
also, we can add a line to README.md to use only python3 for the project